mlflow/utils/requirements_utils.py

```diff
@@ -204,6 +205,8 @@ def _run_command(cmd, timeout_seconds, env=None):
     """
     Runs the specified command. If it exits with non-zero status, `MlflowException` is raised.
     """
+    env = env or os.environ.copy()
+    env[MLFLOW_IN_CAPTURE_MODULE_PROCESS.name] = "true"
```

Comment:
you can pass the environment variable when **_calling_** `_run_command`, not here.